### PR TITLE
Make Wiki.js autostart

### PR DESCRIPTION
**Pull request by rubicae**
Adds systemd service to make wiki.js autostart

### CODE REVIEW

File: ansible/playbooks/wiki/wiki.service [hunk 1]
```diff
@@ -0,0 +1,15 @@
+[Unit]
+Description=Wiki.js Service
+After=network.target
+
+[Service]
+Type=forking
```
Comment (marcidy) [inline ansible/playbooks/wiki/wiki.service]: Everything looks good, just curious about why Type=forking?  Wiki.js docs have it as Type=simple.   

https://docs.requarks.io/install/linux

not saying it's wrong and I assume it works.  I checked the groups and directories mentioned exist in the playbook.

**Review by marcidy — COMMENTED** [review]
Should [playbooks/all.yml](https://github.com/queeriouslabs/infrastructure/blob/wikijs-autostart/ansible/playbooks/all.yml) be updated?

File: ansible/playbooks/wiki/wiki.service (hunk 1 above)
Comment (rubicae) [reply ansible/playbooks/wiki/wiki.service]: Ah, I hadn't seen those docs. Thanks for pointing that out! I think I'll change the service to be more inline with what they suggest.

To answer your question about why I did Type=forking:
I based the service file off of how the service is currently being manually started and stopped.  If you look at the `ExecStart` and `ExecStop` options, you'll see that I'm running `/usr/bin/node wiki [start|stop]` whereas the docs suggest `/usr/bin/node server` for `ExecStart`.  Unlike the `server` program, the `wiki` program is itself a process manager ([PM2](https://pm2.keymetrics.io/)) which would control the `server` process and auto-daemonizes on start, hence the forking type.

Comment (marcidy) [reply ansible/playbooks/wiki/wiki.service]: LGTM

**Review by marcidy — APPROVED** [review]
LGTM

**Pull request merged**